README.md

```diff
@@ -100,335 +143,398 @@ immunopepper make_bg \
+
+*Optional general output files parameters*: Parameters for the files that are output by the software regardless of the filtering method.
+- `--output-count`: Default = ''. Path and name where the intermediate numbers of kmers remaining after each filtering step will be written. If selected, a file will be written containing the number of kmers present after each filtering step. It might slow down the computations. However, it is useful if there is an interest on intermediate filtering steps. The output can be seen in number ### of output section.
+- `--tag-normals`: Default = ''. Name for the normal cohort used for filtering. Needed when there are various normal cohorts. It will be added to the final output name in order to identify against what normal cohort were the samples filtered.
+- `--tag-prefix`: Default = ''. Prefix used for the output files. It is recommended when there are different conditions being studied.
+
+*Optional parameters for normal samples filtering*: Parameters to perform the step 2 of the normal filtering pipeline.
+- `--path-normal-matrix-segm`: Default = None. Path to the matrix of segment expression of kmers in normal samples. This corresponds to output 5 of [output, mode build](#output-files)
+- `--path-normal-matrix-edge`: Default = None. Path to the matrix of junction expression of kmers in normal samples. This corresponds to output 6 of [output, mode build](#output-files)
+- `--n-samples-lim-normal`: Default = None. This is the value for threshold b) in step 2 of normal filtering pipeline. This number will set the number of samples in which we need to see any expression of a specific kmer, i.e. expression > 0, to consider it a normal kmer and exclude it as cancer candidate.
+- `--cohort-expr-support-normal`: Default = None. This is the value for threshold a) in step 2 of normal filtering pipeline. This number corresponds to the normalized expression we need to see in at least one sample in order to consider a kmer as a normal kmer. If a kmer is found with an expression level higher than this threshold in one or more normal samples it will be considered a normal kmer and excluded as cancer candidate.
+
+*Optional parameters for cancer samples filtering*: Parameters to perform the step 2 of the cancer filtering pipeline.
+- `--sample-expr-support-cancer`: #TODO: doesn't have a default. Errors if not provided? This parameter corresponds with the sample specific filtering in the cancer pipeline. The value will correspond to the normalized expression threshold that needs to be reached by each kmer in order to be considered a kmer candidate. Kmers with an expression higher than this threshold in at least one sample will be considered as cancer candidates. #TODO: are samples provided under `--ids_cancer_samples` included in this filtering?
```

Comment:
Default should be None.

Reply:
-> corresponds with to sample specific filtering in the cancer pipeline.

Reply:
This is not correct: Kmers with an expression higher than this threshold in at least one sample will be considered as cancer candidates.
Should be: 
 Kmers with an expression higher or equal than this threshold in the cancer sample of interest will be considered as cancer candidates. Cancer sample of interest samples are provided under `--ids_cancer_samples`.